web/src/data/events.js

```diff
@@ -0,0 +1,85 @@
+import messages from "../common/messages";
+
+/**
+ * Exposes a function to get the details of the next [numEvents] [Event]s.
+ *
+ * Callers are expected to check the [type] of the [Event] and handle rendering.
+ * Other [EventTypes] should be added to the const list. The [Event] class
+ * can also be extended to allow adding additional details (i.e. schedule, pricing,
+ * etc.) And should be handled be the caller to be rendered properly.
+ */
+
+const EventTypes = Object.freeze({
+  DYOS: Symbol("DYOS"),
+});
+
+class Event {
+  constructor(type, title, date, eventSchedule, externalLinks) {
+    this.type = type;
+    this.title = title;
+    this.date = date;
+    this.eventSchedule = eventSchedule;
+    this.externalLinks = externalLinks;
+  }
+}
+
+const defaultDyosSchedule = [
+  { time: "7:30pm - 8:15pm", name: "Building Blocks" },
+  { time: "8:15pm - 9:00pm", name: "Creative Expressions" },
+  { time: "8:45pm - 9:30pm", name: "Welcome to WCS" },
+  { time: "9:00pm - 11:30pm", name: "Social Dancing" },
+];
+
+/**
+ * Hardcoded database of all of the events.
+ *
+ * Keep ordered by date ascending.
```

Comment:
Do we eventually want to delete any of these? Or will we keep every event forever as a record?

Reply:
Eh, that's a future problem